android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/translate/TranslationOperation.kt

```diff
@@ -30,6 +30,11 @@ enum class TranslationOperation {
      */
     FETCH_PAGE_SETTINGS,
 
+    /**
+     * The list of never translate sites the translation engine should fetch.
+     */
+    FETCH_AUTOMATIC_LANGUAGE_SETTINGS,
```

Comment:
Based on the comment, I'm wondering if this name should be something more like `FETCH_NEVER_TRANSLATE_SITES_SETTINGS`?

Reply:
I suspect the comment used here should actually be closer to what you used in https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/translate/TranslationsRuntime.kt#164.

```suggestion
    /**
     * Fetch the user preference on whether to offer, always translate, or never translate for all supported language settings.
     */
    FETCH_AUTOMATIC_LANGUAGE_SETTINGS,
```